### DISCUSSION TITLE
My Page shows a 404 error but it says it has been published

### DISCUSSION THREAD

**Discussion by Daniel-ipymb** [community/community#21569] [Other Feature Feedback, Questions, & Ideas]
<p>I have been trying different options from stack overflow and github but none seems to work. Please can someone help me with this?</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/daniel-ipymb">@Daniel-ipymb</a>, welcome to the GitHub Community! Can you post a link to your site so we can take a look?</p>

**Comment by Daniel-ipymb** [comment]
<p><a href="https://daniel-ipymb.github.io/My_Portfolio/" rel="noopener nofollow ugc">https://daniel-ipymb.github.io/My_Portfolio/</a> (the uploaded page to the lunched website)</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Thanks for that! I think this is happening because of case-sensitivity. It looks like you’ve named your homepage <code>Index.html</code>, but Pages expects the file to be called <code>index.html</code> in order to serve it by default.</p>
<p>Once you rename <code>Index.html</code> to <code>index.html</code> you should stop seeing that 404 error :sparkles:</p>
